packages/demos/stories/Recharts.tsx

```diff
@@ -537,7 +538,7 @@ storiesOf('Demos|Recharts', module)
 					</div>
 				</div>
 
-				<div className="container-fluid container-view">
+				<ClayLayout.ContainerFluid view>
```

Comment:
Same, this would actually now be `xl` because of [defaults](https://github.com/liferay/clay/blob/master/packages/clay-layout/src/ContainerFluid.tsx#L20) which might not be so bad after all 😉